bundles/org.openhab.core.addon/src/main/java/org/openhab/core/addon/AddonInfoListReader.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.addon;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.addon.internal.xml.AddonDiscoveryMethodConverter;
+import org.openhab.core.addon.internal.xml.AddonInfoConverter;
+import org.openhab.core.addon.internal.xml.AddonInfoListConverter;
+import org.openhab.core.addon.internal.xml.AddonInfoXmlResult;
+import org.openhab.core.addon.internal.xml.AddonMatchPropertyConverter;
+import org.openhab.core.config.core.ConfigDescription;
+import org.openhab.core.config.core.ConfigDescriptionParameter;
+import org.openhab.core.config.core.ConfigDescriptionParameterGroup;
+import org.openhab.core.config.core.FilterCriteria;
+import org.openhab.core.config.core.xml.ConfigDescriptionConverter;
+import org.openhab.core.config.core.xml.ConfigDescriptionParameterConverter;
+import org.openhab.core.config.core.xml.ConfigDescriptionParameterGroupConverter;
+import org.openhab.core.config.core.xml.FilterCriteriaConverter;
+import org.openhab.core.config.core.xml.util.NodeAttributes;
+import org.openhab.core.config.core.xml.util.NodeAttributesConverter;
+import org.openhab.core.config.core.xml.util.NodeList;
+import org.openhab.core.config.core.xml.util.NodeListConverter;
+import org.openhab.core.config.core.xml.util.NodeValue;
+import org.openhab.core.config.core.xml.util.NodeValueConverter;
+import org.openhab.core.config.core.xml.util.XmlDocumentReader;
+
+import com.thoughtworks.xstream.XStream;
+
+/**
+ * The {@link AddonInfoListReader} reads XML documents, which contain the {@code binding} XML tag, and converts them to
```

Comment:
```suggestion
 * The {@link AddonInfoListReader} reads XML documents, which contain the {@code addon} XML tag, and converts them to
```
From where did you copy this JavaDoc? Then it might have to be fixed there as well. :-)

Reply:
> From where did you copy this JavaDoc? 

AddonInfoReader

> Then it might have to be fixed there as well. :-)

Done too.